tensorflow/c/c_api.cc

```diff
@@ -81,6 +81,9 @@ extern "C" {
 const char* TF_Version() { return TF_VERSION_STRING; }
 
 // --------------------------------------------------------------------------
+int TF_DataTypeSize(TF_DataType dt) { return tensorflow::DataTypeSize(static_cast<DataType>(dt)); }
```

Comment:
Please format this with the return statement on its own line.  We keep our C++ files to < 80 chars per line.